src/components/View/Bando/BandoNoteAggiornamento.jsx

```diff
@@ -13,9 +13,9 @@ const BandoNoteAggiornamento = ({ content }) => {
   const intl = useIntl();
   return content?.note_aggiornamento ? (
     <div className="mb-4">
-      <h5>
+      <h2 clasName="h5">
```

Comment:
https://iosanita-staging.c3p0.redturtle.it/test-rt-new/bando-di-prova#:~:text=Note%20di%20aggiornamento